types/src/write_set.rs

```diff
@@ -390,7 +382,7 @@ impl TransactionWrite for WriteOp {
             None => Self::legacy_deletion(),
             Some(state_value) => {
                 let (metadata, data) = state_value.unpack();
-                Self::Creation { data, metadata }
```

Comment:
Why are we replacing Creation with Modification?

Reply:
see the answer above?